apisix/plugins/ai.lua

```diff
@@ -116,7 +117,26 @@ end
 
 
 local pool_opt
-local function ai_balancer_run(route)
+local function ai_http_balancer_phase()
+    local api_ctx = ngx.ctx.api_ctx
+    if not api_ctx then
+        core.log.error("invalid api_ctx")
+        return core.response.exit(500)
+    end
+
+    if is_apisix_or then
+        local ok, err = response.skip_header_filter_by_lua()
```

Comment:
Only the current request skips the ***_filter, all right?

Reply:
no, it effects global